french-soup/index.html

```diff
@@ -3,19 +3,71 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>French Onion Soup Recipe</title>
-    <!-- don't forget to link your styles -->
+    <title>Apple Pie Recipe</title>
+    <link rel="stylesheet" href="styles/style.css">
   </head>
+
+
   <body>
-    French Onion Soup We have been trying French onion soup in restaurants for years and my family and friends agree
-    none can compare to my recipe for taste and simplicity of preparation. Ingredients 1/2 cup unsalted butter 2
-    tablespoons olive oil 4 cups sliced onions 4 (10.5 ounce) cans beef broth 2 tablespoons dry sherry (optional) 1
-    teaspoon dried thyme salt and pepper to taste 4 slices French bread 4 slices provolone cheese 2 slices Swiss cheese,
-    diced 1/4 cup grated Parmesan cheese Directions Melt butter with olive oil in an 8 quart stock pot on medium heat.
-    Add onions and continually stir until tender and translucent. Do not brown the onions. Add beef broth, sherry and
-    thyme. Season with salt and pepper, and simmer for 30 minutes. Heat the oven broiler. Ladle soup into oven safe
-    serving bowls and place one slice of bread on top of each (bread may be broken into pieces if you prefer). Layer
-    each slice of bread with a slice of provolone, 1/2 slice diced Swiss and 1 tablespoon Parmesan cheese. Place bowls
-    on cookie sheet and broil in the preheated oven until cheese bubbles and browns slightly.
+
+    <header>
+      <h1>French Onion Soup</h1>
+    </header>
+
+
+   <div class="history">
+    <p>
+      We have been trying French onion soup in restaurants for years and my family and friends agree none can compare to my recipe for taste and simplicity of preparation.
+    </p>
+      <img src="/french-soup/images/recipe-info.png" alt="recipeTime">
+    </div>
+
+
+
+    <div class="ingredients">
+
+      <p>Ingredients</p>
+
+    </div>
+
+      <div class="items">
+
+       <ol>
+        <p>1/2 cup unsalted butter</p>
```

Comment:
When you have an ordered list `<ol>` or unordered list `<ul>` you should use the list item tag `<li>` inside, instead of using the paragraph tag `<p>` and then you can style it in the `css` file